packages/react-components/src/components/Dialpad/Dialpad.test.tsx

```diff
@@ -1,14 +1,13 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
```

Comment:
Just curious why this was necessary

Reply:
Typescript isn't smart enough to know that:
```
var a: someType | undefined;

expect(a).toBeTruthy();
expect(a.something).toBe(...); // Typescript complains here that a might be undefined, but it actually can't be undefined because of the previous `expect`

// so I instead do:
expect(a).toBeTruthy();
expect(a!.something).toBe(...); // notice the ! after the a, this is the non-null assertion, which by default we disallow as its should be used more sparingly
```